radixdlt/babylon-wallet-android
# [ABW-3777] - Fix error when viewing Hidden Assets

[Jira Ticket](https://radixdlt.atlassian.net/browse/ABW-3777)

## Description
This PR fixes the error message toast and the infinite loading on hidden assets screen. The failing gateway call for a resource from a different network caused the error and it was fixed in the latest Sargon version.


## How to test

1. Hide some assets on stokenet
2. Change network to mainnet
3. Open the hidden assets screen and make sure there are no errors
